examples/chef/common/chef-laundry-washer-mode.cpp

```diff
@@ -94,6 +95,61 @@ void LaundryWasherMode::Shutdown()
     }
 }
 
+chip::Protocols::InteractionModel::Status chefLaundryWasherModeWriteCallback(chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                                                             const EmberAfAttributeMetadata * attributeMetadata,
+                                                                             uint8_t * buffer)
+{
+    VerifyOrDie(endpointId == 1); // this cluster is only enabled for endpoint 1
+    VerifyOrDie(gLaundryWasherModeInstance != nullptr);
```

Comment:
why die? This method can fail with a status code? Why not log and return a failure instead?

Reply:
done. return error